aws.yaml

```
@@ -331,8 +331,58 @@
       version: 3.10.1
     - name: etcd
       version: 3.3.17
-- version: 9.0.0
+- version: 9.0.1
   state: active
+  date: 2020-03-17T19:00:00Z
+  apps:
+    - app: cert-exporter
+      version: 1.2.1
+    - app: chart-operator
+      version: 0.12.1
+    - app: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.3
+    - app: coredns
+      componentVersion: 1.6.5
+      version: 1.1.7
```

There was a new coredns release https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v118-2020-03-20
```suggestion
      version: 1.1.8
```